packages/experimental/Shimmer/src/Shimmer.win32.tsx

```diff
@@ -0,0 +1,104 @@
+/** @jsx withSlots */
+import { compose, mergeProps, UseSlots } from '@fluentui-react-native/framework';
+import { View } from 'react-native';
+import { ClipPath, Defs, LinearGradient, Path, Rect, Stop, Svg } from 'react-native-svg';
+import { styleSettings } from './Shimmer.styling.win32';
+import { shimmerName, ShimmerProps } from './Shimmer.types';
+import { ShimmerType } from './Shimmer.types.win32';
+import { RCTNativeAnimatedShimmer } from './consts.win32';
+import { convertRectToSvgPath, convertCircleToSvgPath } from './SvgShapeToPath';
+import { withSlots } from '@fluentui-react-native/framework';
+
+/** Absolute positioning is used to overlay the clipping mask on top of the shimmer wave. */
+const clippingMask: React.FunctionComponent<ShimmerType['slotProps']['clippingMask']> = (
```

Comment:
Wait, why couldn't this just be ShimmerType.slotProps.clippingMask? Same for wave below

Reply:
Or maybe define ShimmerType.slotProps.clippingMask as a separate type that both use?

Reply:
`Cannot access 'ShimmerType.slotProps' because 'ShimmerType' is a type, but not a namespace. ` tbh this confused me too.  I'll use the slot prop types directly, I didn't really like depending on it being "coordinated"